Mathlib/Algebra/Order/Antidiag/Finsupp.lean

```diff
@@ -1,167 +1,82 @@
 /-
 Copyright (c) 2023 Antoine Chambert-Loir and María Inés de Frutos-Fernández. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Antoine Chambert-Loir, María Inés de Frutos-Fernández, Eric Wieser, Bhavik Mehta
+Authors: Antoine Chambert-Loir, María Inés de Frutos-Fernández, Eric Wieser, Bhavik Mehta,
+  Yaël Dillies
 -/
 import Mathlib.Algebra.Order.Antidiag.Pi
 import Mathlib.Data.Finsupp.Basic
 
 /-!
-# Partial HasAntidiagonal for functions with finite support
+# Antidiagonal of finitely supported functions as finsets
 
-For an `AddCommMonoid` `μ`,
-`Finset.HasAntidiagonal μ` provides a function `antidiagonal : μ → Finset (μ × μ)`
-which maps `n : μ` to a `Finset` of pairs `(a, b)` such that `a + b = n`.
+This file provides the finset of finitely functions summing to a specific value on a finset. Such
```

Comment:
provides... "the definition and the main properties of", or something like this? Also, can you point to the main declarations of the file?